e2e/rbd_helper.go

```diff
@@ -547,8 +547,21 @@ func validateEncryptedImage(f *framework.Framework, rbdImageSpec string, app *v1
 		return fmt.Errorf("%v not equal to encrypted", encryptedState)
 	}
 
-	volumeMountPath := app.Spec.Containers[0].VolumeMounts[0].MountPath
-	mountType, err := getMountType(app.Name, app.Namespace, volumeMountPath, f)
+	pod, err := f.ClientSet.CoreV1().Pods(f.UniqueName).Get(context.TODO(), appName, metav1.GetOptions{})
+	if err != nil {
+		e2elog.Logf("error getting pod %q in namespace %q", appName, f.UniqueName)
```

Comment:
@pkalever do we need this here? I doubt thats the case.

Reply:
@humblec you mean just delete the logline, right?

Reply:
Done now!

Reply:
yep. @pkalever

Reply:
Thanks!